docs/pages/choose-an-edition/teleport-enterprise/hsm.mdx

```diff
@@ -324,23 +324,22 @@ route any traffic to the auth server where the HSM is currently being added.
 
 When adding a new HSM to an existing Teleport cluster, or adding a new
 HSM-connected Auth server to an HA Teleport cluster, you will need to rotate all
-Certificate Authorities in order for new certificates to by issued and
+Certificate Authorities in order for new certificates to be issued and
 trusted.
 
 `tctl status` will print a warning if CA rotation is required:
 ```code
 $ tctl status
 WARNING: One or more auth servers has a newly added or removed HSM or KMS configured. You should not route traffic to that server until a CA rotation has been completed.
 Cluster      cluster-one
-Version      13.0.0-alpha.1
+Version      (=teleport.version=)
 host CA      never updated
 user CA      never updated
 db CA        never updated
 openssh CA   never updated
 jwt CA       never updated
 saml_idp CA  never updated
-CA pin       sha256:29afff5a1c1231375525b8a9b98ea7a44a760b7e65de3ca2e04d72db39b1672b
-CA pin       sha256:e758c8f0f6cd95116d5da8171e0ff4adfa99dab3b1f171bfe854070884955524
```

Comment:
Pretty sure we were intentionally showing multiple pins here.

Reply:
restored the 2nd one.